example/eclipselink/src/test/kotlin/com/linecorp/kotlinjdsl/example/eclipselink/select/SelectExample.kt

```diff
@@ -287,4 +287,52 @@ class SelectExample : WithAssertions {
         // then
         assertThat(actual).isEqualTo(listOf(7L))
     }
+
+    @Test
+    fun `employee who doesn't have a nickname`() {
+        // when
+        val expected = "If the replacement value is null, this value is extracted"
```

Comment:
It doesn't need an example for `coalesce`. Please remove this, because it forces us to create an example for every function.

Reply:
There are errors that cannot be known unless you run the actual example. Would it be better to remove it? I also ran it and found that a space is needed before coalesce.

Reply:
I re-checked the normal operation of the query and deleted the example codes because I agree with your opinion. Thank you